### ISSUE TITLE
Incorrect named ranges after adding rows in a table

### ISSUE THREAD

**Issue by irvalchev** [EPPlusSoftware/EPPlus#871]
I have noticed that there is an issue with the named ranges when a row is added in a table. The new rows are not considered in the named ranges that originally should contain them. Please refer on the example below (check range componentTypes). Also here is code to reproduce it:
```
            string folder = @"C:\";
            string path = folder + "test.xlsx";
            using (var package = new ExcelPackage(path))
            {
                var table = package.Workbook.Worksheets.SelectMany(x => x.Tables).Single(x => x.Name == "TblComponentTypes");
                table.AddRow(2);
                table.WorkSheet.Cells[8, 1].Value = "TST";
                table.WorkSheet.Cells[9, 1].Value = "TST 2";

                package.SaveAs(folder + "test_updated.xlsx");
            }
```

This is the original named range declaration: =TblComponentTypes[Failure Rate Category]
And here the one after running the code: ='Failure Rates'!$A$2:$A$7

[test_updated.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/11418840/test_updated.xlsx)
[test.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/11418842/test.xlsx)

**Comment by JanKallman** [comment]
Thanks, we'll have a look at this.

**Comment by JanKallman** [comment]
Yes, the reason it work is that if you don't access the pivot tables they will not be updated. If you just access the one of the pivot tables they will be loaded and updated on save. Just uncomment this row will cause the workbook to become corrupt (or cause an exception on save if you use 6.2.3)...
`ExcelPivotTable pivottable = wkMonthlyDetail.PivotTables["ptMonthlyDetail"];`
I will see if I can find the real cause with the second pivot chart as mentioned before .

**Comment by JanKallman** [comment]
This is a bug. EPPlus changes the table address to a normal address, causing the update to be incorrect. I will provide a fix.

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.2.4
